### ISSUE TITLE
Error when loading lora weights

### ISSUE THREAD

**Issue by kelsey-um** [tloen/alpaca-lora#600]
I get the following error when I execute the code in generate.py.

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
[<ipython-input-4-e069152edb75>](https://localhost:8080/#) in <cell line: 41>()
     46         device_map="auto",
     47     )
---> 48     model = PeftModel.from_pretrained(
     49         model,
     50         lora_weights,

[/usr/local/lib/python3.10/dist-packages/peft/peft_model.py](https://localhost:8080/#) in from_pretrained(cls, model, model_id, adapter_name, is_trainable, config, **kwargs)
    276         # load the config
    277         if config is None:
--> 278             config = PEFT_TYPE_TO_CONFIG_MAPPING[
    279                 PeftConfig._get_peft_type(
    280                     model_id,

[/usr/local/lib/python3.10/dist-packages/peft/config.py](https://localhost:8080/#) in from_pretrained(cls, pretrained_model_name_or_path, subfolder, **kwargs)
    132 
    133         kwargs = {**class_kwargs, **loaded_attributes}
--> 134         config = config_cls(**kwargs)
    135         return config
    136 

TypeError: LoraConfig.__init__() got an unexpected keyword argument 'enable_lora'

---------------------------------------------------------------------------

Part of code with error:
`model = PeftModel.from_pretrained(model,
        lora_weights,
        torch_dtype=torch.float16,)`

I am using _huggyllama/llama-7b_ as the base model and _tloen/alpaca-lora-7b_ for the lora weights. 

Any help would be greatly appreciated!

**Comment by kelsey-um** [comment]
Issue solved in https://github.com/huggingface/peft/issues/1087

**Comment by Nimisha-Pabbichetty** [comment]
@kelsey-um I followed the same procedure as you and was able to get the generate.py script working. However, with the finetune.py script I'm facing the following error:
![image](https://github.com/tloen/alpaca-lora/assets/60477893/973c4308-0ff9-4407-9126-c54256894a3b)
Any idea how to solve it? It seems to be because the load_in_8bit parameter is set to True but if I set it to False then even a GPU with vRAM of 44Gb is not enough to train the model.
